frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/change_cover_popover.dart

```diff
@@ -279,14 +322,66 @@ class _ChangeCoverPopoverState extends State<ChangeCoverPopover> {
   }
 }
 
+class DeleteImageAlertDialog extends StatelessWidget {
+  const DeleteImageAlertDialog({
+    Key? key,
+    required this.onSubmit,
+  }) : super(key: key);
+
+  final Function() onSubmit;
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      title: FlowyText.semibold(
+        "Image is used in cover",
+        fontSize: 20,
+        color: Theme.of(context).colorScheme.tertiary,
+      ),
+      content: Container(
+        constraints: const BoxConstraints(minHeight: 100),
+        padding: const EdgeInsets.symmetric(
+          vertical: 20,
+        ),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: const [
+            Text("It will be removed from cover after it is deleted."),
```

Comment:
Please add this text to the translation file.

Reply:
ok